src/bindings/python/tests/test_frontend/test_frontend_onnx_editor.py

```diff
@@ -91,6 +101,23 @@ def create_test_onnx_models():
     models["input_model_2.onnx"] = make_model(graph, producer_name="ONNX Importer",
                                               opset_imports=[onnx.helper.make_opsetid("", 13)])
 
+    # Input model 3
+    add_2 = onnx.helper.make_node("Add", inputs=["in1", "in2"], outputs=["out1"], name="onnx_add_op")
+    relu_2 = onnx.helper.make_node("Relu", inputs=["in2"], outputs=["out2"])
+
+    input_tensors = [
+        make_tensor_value_info("in1", onnx.TensorProto.FLOAT, (2, 2)),
+        make_tensor_value_info("in2", onnx.TensorProto.FLOAT, (2, 2)),
+    ]
+    output_tensors = [
+        make_tensor_value_info("out1", onnx.TensorProto.FLOAT, (1, 2)),
+        make_tensor_value_info("out1", onnx.TensorProto.FLOAT, (2, 2)),
```

Comment:
Can you run ONNX checker on this model? `out` is defined twice but each time with a different shape. I'm not sure that's allowed by the standard.

Reply:
It was a typo, thank you for catching it. Set both shapes to (2, 2).